frontend/src/queries/Project.ts

```diff
@@ -37,6 +53,26 @@ export function useProjectsQuery(): UseQueryReturnType<Project[], Error> {
     });
 }
 
+export function useGroupsQuery(
+    projectId: Ref<number | undefined>
+): UseQueryReturnType<Group[], Error> {
+    return useQuery<Group[], Error>({
+        queryKey: computed(() => GROUPS_QUERY_KEY(projectId.value!)),
+        queryFn: () => getGroups(projectId.value!),
+        enabled: computed(() => projectId.value !== undefined),
+    });
+}
```

Comment:
misschien nog wachten met mergen totdat Bram's query refactor is doorgevoerd, dan kan je hier `useProjectGroupsQuery` gebruiken.